### ISSUE TITLE
Trailing Stop Loss - Hyperopt Problems 

### ISSUE THREAD

**Issue by Prossi79** [freqtrade/freqtrade#2620]
## Step 1: Have you search for this issue before posting it?

If you have discovered a bug in the bot, please [search our issue tracker](https://github.com/freqtrade/freqtrade/issues?q=is%3Aissue). 
If it hasn't been reported, please create a new issue.

## Step 2: Describe your environment

  * Operating system: Ubuntu 18.04
  * Python Version: 3.7.4
  * CCXT version: ccxt==1.20.22
  * Branch: Develop
  * Last Commit ID: 8dd9b5c6fba973b98defc0f9a9e2e3c01af735dd

  
## Step 3: Describe the problem:

Just played around with the new hyperopt feature - trailing ... I hyperopted just the roi stoploss and trailing spaces of a strategy, following result:

ROI table:
{0: 0.15153, 36: 0.05431, 80: 0.01165, 128: 0}
Stoploss:
{'stoploss': -0.22779}
Trailing stop:
{   'trailing_only_offset_is_reached': False,
    'trailing_stop': True,
    'trailing_stop_positive': 0.34424,
    'trailing_stop_positive_offset': 0.03257}

### Relevant code exceptions or logs:

  ```
 2019-12-05 15:32:13,404 - freqtrade - ERROR - The config trailing_stop_positive_offset needs to be greater than trailing_stop_positive in your config.
  ```

**Comment by hroff-1902** [comment]
Duplicated to #2617, @xmatthias was faster. 😆 

I'll close it as duplicated if you don't mind.

**Comment by hroff-1902** [comment]
@Prossi79 the fix is merged into develop: #2642, pls verify
